src/fw/drivers/nrf5/qspi.c

```diff
@@ -20,70 +19,64 @@
 
 #define NRF5_COMPATIBLE
 #include <mcu.h>
+#include <nrfx_qspi.h>
 
 #include "FreeRTOS.h"
 #include "semphr.h"
 
-#include <nrfx_qspi.h>
-
 /* nRF5's QSPI controller is different enough from STM32's that we
  * reimplement qspi_flash.c, not stm32/qspi.c.  */
 
-//! Address value which signifies no address being sent
-#define QSPI_ADDR_NO_ADDR (UINT32_MAX)
-//! Word size for DMA reads
-#define QSPI_DMA_READ_WORD_SIZE (4)
+#define FLASH_RESET_WORD_VALUE (0xffffffff)
 
+static void prv_use(QSPIPort *dev) { dev->state->use_count++; }
 
-static void prv_use(QSPIPort *dev) {
-  dev->state->use_count++;
-}
```

Comment:
is this style common elsewhere in PebbleOS?  I don't remember seeing it elsewhere

Reply:
we have clang-format enabled (and enforced). It may still require some tuning? @Hexxeh

Reply:
fine by me, as long as this is intentional

Reply:
I see some weird formatting TBH, will check if it can be tuned a bit when I get stuck with some nasty bug.